src/TicTacToe.js

```diff
@@ -12,35 +12,22 @@ var TicTacToe = function() {
 
 TicTacToe.prototype.menu = function() {
   this.ui = new UI();
-  this.ui.onButtonClicked(TicTacToe.prototype.determineGameType);
+  this.ui.onButtonClicked(TicTacToe.prototype.determineGameType, this.ui);
```

Comment:
A little weird, but after working with you on Friday this is probably the best solution 👍 Would be nice if we could figure something else out down the line!